nixos/modules/profiles/hardened.nix

```
@@ -110,6 +110,9 @@ in
     # Disable ftrace debugging
     boot.kernel.sysctl."kernel.ftrace_enabled" = mkDefault false;
 
+    # Disable io_uring, a large source of security vulnerabilities
```

```suggestion
    # Disable io_uring, a large source of security vulnerabilities
    # https://security.googleblog.com/2023/06/learnings-from-kctf-vrps-42-linux.html
```
At least we should put a link to the source of the suggestion here.